paddle/fluid/operators/mkldnn/mul_mkldnn_op.cc

```diff
@@ -207,6 +207,10 @@ class QuantMulPrimitiveFactory : public MulPrimitiveFactory<XT, YT, OT> {
     int y_num_col_dims = ctx.Attr<int>("y_num_col_dims");
     auto scale_y = ctx.Attr<std::vector<float>>("scale_y");
 
+    // TODO(minghui) : to make int8 mul op more general.
+    bool enforce = std::is_same<YT, float>::value;
+    PADDLE_ENFORCE(enforce == true, "Input(Y) supposed to be float type.");
```

Comment:
I think we can add a little bit more explanation in enforce to illustrate the reason why Input(Y) supposed to be float type.

Reply:
done!